File: templates/python/power_system_deep_calendar_grid.py
"""power_system_deep_calendar_grid: 电力系统深化日历网格（calendar-grid 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='power system analysis: calendar grid'):
    return make_template_figure('calendar_grid', seed=3619, title=title, domain='power system analysis', topic='calendar grid')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/power_system_deep_calendar_grid.m
function fig = power_system_deep_calendar_grid()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('calendar_grid', 3619, 'power system analysis: calendar grid', 'power system analysis', 'calendar grid');
end
